--- node_modules/simdjson/simdjson/src/simdjson.h
+++ node_modules/simdjson/simdjson/src/simdjson.h
@@ -8814,10 +8814,10 @@
   // Checks if there are any characters that need escaping
   simdjson_really_inline bool has_escaping_char(size_t idx) {
     // do 8 chars at a time
-    if(needs_escaping[uint8_t(unescaped[i])] | needs_escaping[uint8_t(unescaped[i+1])]
-      | needs_escaping[uint8_t(unescaped[i+2])] | needs_escaping[uint8_t(unescaped[i+3])]
-      | needs_escaping[uint8_t(unescaped[i+4])] | needs_escaping[uint8_t(unescaped[i+5])]
-      | needs_escaping[uint8_t(unescaped[i+6])] | needs_escaping[uint8_t(unescaped[i+7])]) {
+    if(needs_escaping[uint8_t(unescaped[i])] || needs_escaping[uint8_t(unescaped[i+1])]
+      || needs_escaping[uint8_t(unescaped[i+2])] || needs_escaping[uint8_t(unescaped[i+3])]
+      || needs_escaping[uint8_t(unescaped[i+4])] || needs_escaping[uint8_t(unescaped[i+5])]
+      || needs_escaping[uint8_t(unescaped[i+6])] || needs_escaping[uint8_t(unescaped[i+7])]) {
       return true;
     }
     return false;
   }
 }
